resources/definitions/fdmprinter.def.json

```
@@ -2755,6 +2755,22 @@
                     "default_value": 0.940860215,
                     "enabled": false
                 },
+                "max_degradation_temperature":
+                {
+                    "label": "Max degradation temperature",
```

We normally use word-caps for setting labels.
```suggestion
                    "label": "Max Degradation Temperature",
```